test-cases/longevity/longevity-twcs-3h.yaml

```diff
@@ -1,13 +1,13 @@
 test_duration: 210
 bench_run: true
 
-stress_cmd: ["scylla-bench -workload=timeseries -mode=write -replication-factor=3 -partition-count=400 -clustering-row-count=10000000 -clustering-row-size=200 -concurrency=100 -rows-per-request=100 -start-timestamp=SET_WRITE_TIMESTAMP -connection-count 100 -max-rate 50000 --timeout 120s -duration=170m"]
+stress_cmd: ["scylla-bench -workload=timeseries -mode=write -replication-factor=3 -partition-count=4000 -clustering-row-count=100000 -clustering-row-size=200 -concurrency=50 -rows-per-request=100 -start-timestamp=SET_WRITE_TIMESTAMP -connection-count 50 -max-rate 50000 --timeout 120s -duration=170m"]
```

Comment:
I think that for timeseries, the typical use case won't have such a big row count.
Maybe we misinterpret the scylla-bench usage and we need to keep the partitions small (for example 10 rows that are reporting metrics of a sensor in particular time).